src/constants.rs

```diff
@@ -1 +1,5 @@
-pub const N_BITS_CHALLENGE: usize = 250;
+// used for challenge r, so when computing r^2 it does not overflow the field
```

Comment:
Right! I'm seeing now at https://eprint.iacr.org/2023/969.pdf that they only mention (page 6) the 250 bits output of the hash of the instances (as we already have), but there is no mention to the 128 bits for the challenge `r`.
On the other hand, in https://github.com/microsoft/Nova/blob/main/src/constants.rs#L1 they define the 128 bits for the challenge. I'm thinking that maybe is just to have less constraints when computing the scalar mul with the challenge (?).
I'll ask Srinath about it to learn, but meanwhile I will remove the 128 line since it is not used yet in the code.